modules/pingsource-remove-kn.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_sources/serverless-pingsource.adoc
+
+[id="pingsource-remove-kn_{context}"]
+= Remove a ping source using the Knative CLI
+
+The following procedure describes how to remove a ping source using the `kn` CLI.
```

Comment:
It actually kind of doesn't so I've removed this.
I think the idea originally was deleting it as a clean up step if it was created to use with the event source, but it's not required that a service is deleted just because an event source that was connected to it is afaik, so I've removed that step. 

I've also removed it for api server source but any others I'll clean up another time because I don't want to go too far down the rabbit hole with this, and the step to create the service could probably be removed altogether from a lot of procedures or added as a prereq instead. I'll come back to this when I have more time for doc improvements.